docs/source/user-guide/vision/quickstarts/plot_detection_tutorial.py

```diff
@@ -199,7 +158,7 @@ def show(imgs):
 norm_layer = partial(nn.BatchNorm2d, eps=0.001, momentum=0.03)
 
 model.head.classification_head = SSDLiteClassificationHead(in_channels, num_anchors, 2, norm_layer)
-model.to(device)
+_ = model.to(device)
```

Comment:
we need this?

Reply:
yes, if not it writes a long output of the model which is not interesting. 
Can also change to "model = ..." instead